.github/workflows/commitlint.yml

```diff
@@ -7,4 +7,14 @@ on:
 
 jobs:
   commitlint:
-    uses: openedx/.github/.github/workflows/commitlint.yml@master
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: remove tsconfig.json # see issue https://github.com/conventional-changelog/commitlint/issues/3256
+        run: |
+          rm tsconfig.json
```

Comment:
We're not going to need this, going forward.  See this comment on a Paragon PR: https://github.com/openedx/paragon/pull/2867#discussion_r1406343198